### PR TITLE
Updated .gitignore

### PR DESCRIPTION
**Pull request by risdenk**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,29 +1,169 @@
...
+Temporary Items
+.apdisk
+### Vagrant template
 .vagrant/
-vagrant/rpms/
+### Java template
+*.class
 
-dependency-reduced-pom.xml
-twitter4j.properties
```
Comment (kruthar) [inline .gitignore:17]: twitter4j.properties needs to be included so people don't accidently commit their twitter app secrets and keys

Comment (risdenk) [inline .gitignore:17]: Fixed.

File: .gitignore [hunk 2]
```diff
@@ -1,29 +1,169 @@
...
+
+# Directories potentially created on remote AFP share
+.AppleDB
+.AppleDesktop
+Network Trash Folder
+Temporary Items
+.apdisk
+### Vagrant template
 .vagrant/
-vagrant/rpms/
```
Comment (kruthar) [inline .gitignore:14]: the vagrant/rpms/ dir needs to be added, this is a local cache of rpms used to boot the vagrant machines, and the contents should not be commited

Comment (risdenk) [inline .gitignore:14]: Fixed.

File: .gitignore [hunk 3]
```diff
@@ -1,29 +1,169 @@
...
+
+# Mongo Explorer plugin:
+# .idea/mongoSettings.xml
+
+## File-based project format:
+*.ipr
 *.iws
 
-# Maven
-log/
```
Comment (kruthar) [inline .gitignore:10]: the 'log/' dir is not specifically added anywhere although there may be coverage with 'logs' and *.log, is this an issue?

Comment (risdenk) [inline .gitignore:10]: ha was just fixing that.

**Comment by risdenk** [comment]
This will cause some new files to need to be removed. I'll address that in PR #62

File: .gitignore (hunk 3 above)
Comment (risdenk) [inline .gitignore:10]: Fixed.

**Comment by kruthar** [comment]
:+1:

**Pull request merged**